### PR TITLE
TAS-83-nav-bar

### PR DESCRIPTION
**Pull request by adrastaea**
- Blocked by merge of db PRs - 

# Description
Adds a nav bar to navigate to a series of bare bones pages

### CODE REVIEW

**Comment by csrubin** [comment]
Please configure pre-commit hooks (and never worry about formatting again! 😁)

File: how2meet/ui/frontend.py [hunk 1]
```diff
@@ -1,40 +1,124 @@
...
+            with ui.row():
+                ui.input("Description")
+            with ui.row().classes("w-half"):
+                with ui.column():
+                    ui.button("Add guest", on_click=add_guest)
+
+        ui.button("Save", on_click=lambda: ui.notify("Event details would be written to DB here"))
+        ui.button("Back", on_click=lambda: ui.open("/"))
+
+@ui.page("/events")
```
Comment (csrubin) [inline how2meet/ui/frontend.py:82]: fyi this is what it looks like when you don't have a working db lol. What does it look like for you?

Screenshot 2024-01-03 at 10 47 55 PM

**Comment by csrubin** [comment]
Looks pretty decent overall! I will say, I'm not a fan of the navbar items changing position depending on what page you're on. I would prefer for all possible pages to always be visible, with the one that is currently _active_ has some sort of visual emphasis.

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```
Comment (csrubin) [inline .vscode/launch.json:2]: Fancy json that accepts comments? what is this witchcraft?

File: how2meet/ui/frontend.py [hunk 3]
```diff
@@ -1,40 +1,124 @@
...
+def events():
+    """Master list of events in database
```
Comment (csrubin) [inline how2meet/ui/frontend.py:84]: "master" 🤔

File: how2meet/ui/frontend.py (hunk 1 above)
Comment (adrastaea) [reply how2meet/ui/frontend.py:82]: That's what it looks like to me. I didn't make the db work for this PR. Out of scope

File: .vscode/launch.json (hunk 2 above)
Comment (adrastaea) [reply .vscode/launch.json:2]: default vscode thing idk

**Pull request closed without merging**